src/Api/AbstractApi.php

```diff
@@ -36,7 +36,7 @@ abstract class AbstractApi
      *
      * @var string
      */
-    private const URI_PREFIX = '/api/v4/';
+    protected const URI_PREFIX = '/api/v4/';
```

Comment:
Do you really need to change this? Since you're using static now, it should automatically use the one from the child class, no? 🤔

Reply:
In PHP `private` means a sub-class cannot extend/override it.

Reply:
why does it need to be overwritten? the changes you want don't need to re-use this?

Reply:
@GrahamCampbell, GraphQL is _versionless_. The URL prefix is `/api` instead of `/api/v4`, so `AbstractApi::prepareUriP()`, needs to get `/api` when subclassed as `\Gitlab\Api\GraphQL`